### PR TITLE
translate: identity service

### PR DESCRIPTION
**Pull request by ahmeterenodaci**
## How has this been tested?

- [ ] Prod
- [ ] Development

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] Added all translations, no hard coded text. 
- [ ] Updated the UI (if needed).
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: apps/web/src/language-data/IdentityService/index.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  tr,
+  en,
+};
+
+export interface ContractServiceResource {
+  Role: string;
+  Roles: string;
+  "Role.New": string;
+  "Role.Edit": string;
+  "Role.Description": string;
```
Comment (a0m0rajab) [inline apps/web/src/language-data/IdentityService/index.ts:16]: can we have this as the next: 
```js
NewWord: string
Role: string
```

and use it like: 
```
something = `${NewWord+ role}`
```

Comment (ertugrulcan-ays) [reply apps/web/src/language-data/IdentityService/index.ts:16]: Did you consider what will happen when other languages involve this?

Comment (ahmeterenodaci) [reply apps/web/src/language-data/IdentityService/index.ts:16]: There might be a language where adjective (or whatever its called I'm not sure) comes after the word. Like: role + new. Or there might be other exceptions we are not aware. 

@a0m0rajab

**Comment by ahmeterenodaci** [comment]
I forgot to add some translations.

File: apps/web/src/language-data/IdentityService/index.ts (hunk 1 above)
Comment (a0m0rajab) [reply apps/web/src/language-data/IdentityService/index.ts:16]: @ertugrulcan-ays @ahmeterenodaci  I thought about that we could have a structure flag in the backend that change provide us with the related info.

File: apps/web/src/app/[lang]/app/[type]/dashboard/[domain]/layout.tsx [hunk 2]
```diff
@@ -29,13 +29,13 @@ export default function Layout({ children, params }: LayoutProps) {
   useEffect(() => {
     const tempNavbarItems = Object.entries(dataConfig[params.domain])
       .filter(([e]) => e !== "displayName" && e !== "default")
-      .map(([key, value]: [string, any]) => {
+      .map(([key]: [string, unknown]) => {
```
Comment (a0m0rajab) [inline apps/web/src/app/[lang]/app/[type]/dashboard/[domain]/layout.tsx:32]: if we removed the value, should not we remove it's type as well?

Comment (a0m0rajab) [reply apps/web/src/app/[lang]/app/[type]/dashboard/[domain]/layout.tsx:32]:
```suggestion
      .map(([key]: [string]) => {
```

**Review by a0m0rajab — APPROVED** [review]

File: apps/web/src/app/[lang]/app/[type]/dashboard/[domain]/layout.tsx (hunk 2 above)
Comment (ahmeterenodaci) [reply apps/web/src/app/[lang]/app/[type]/dashboard/[domain]/layout.tsx:32]: It gives error.

**Pull request merged**